hello-baremetal/hello.c

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
This one is not a linux program, but in there to have a counterpart for the Linux program? So this should be part of apps-baremetal. Essentially it's a "print everything you get", this could be useful for other tests as well.
Unless we want to keep this PR as work in progress? Then it probably can stay here until the receive side is fixed.

Reply:
I also thought about keeping the baremetal there, but since it is just a unit test that makes sense with the Host Tile running Linux I was not sure the best place to keep it. What are your thoughts on this?